tests/reporting/test_reporting.py

```diff
@@ -1,108 +1,442 @@
 from ethereum.tools import tester
+from ethereum.tools.tester import TransactionFailed
+from pytest import fixture, mark, lazy_fixture, raises
 from datetime import timedelta
 
 tester.STARTGAS = long(6.7 * 10**6)
 
-def test_reporting(contractsFixture):
-    # seed legacy rep contract
-    legacyRepContract = contractsFixture.contracts['legacyRepContract']
+@fixture(scope="session")
+def reportingSnapshot(sessionFixture):
```

Comment:
Style: For test files, I tend to prefer putting the tests themselves at the top, and dependencies and helper methods at the bottom.  The idea is to make it easy for someone reviewing test failures to see the tests first and only dig into the implementation details if necessary (scroll down).